airbyte-integrations/connectors/destination-csv/src/main/java/io/airbyte/integrations/destination/csv/CsvDestination.java

```diff
@@ -192,7 +200,10 @@ protected void close(boolean hasFailed) throws IOException {
       if (!hasFailed) {
         for (final WriteConfig writeConfig : writeConfigs.values()) {
           Files.move(writeConfig.getTmpPath(), writeConfig.getFinalPath(), StandardCopyOption.REPLACE_EXISTING);
+          LOGGER.info(String.format("File output: %s", writeConfig.getFinalPath()));
         }
+      } else {
+        LOGGER.error("Failed to output files in destination");
```

Comment:
I don't like that the error is a log line instead of a sync failure

Reply:
Can we make if fail if no file was written?

Reply:
Sorry, in this PR I was mainly focusing on redefining the destination path taken as input only.

I added some logs here and there as I was hoping to catch some reasons why sometimes it would terminate without writing anything as reported multiple times by users...

However, you are right, we probably should fail instead to be less misleading!